### PR TITLE
LPS-194421

### PR DESCRIPTION
**Pull request by CaiocSiqueira**
https://liferay.atlassian.net/browse/LPS-194421

### CODE REVIEW

**Comment by CaiocSiqueira** [comment]
ci:test:sf

File: test.properties [hunk 1]
```diff
@@ -7510,6 +7510,11 @@
     # Publications
     #
 
+    test.batch.class.names.excludes[publications]=\
+        **/apps/change-tracking/change-tracking-test/src,\
+        /service/test/CTCollectionServiceTest.java,\
+        /testIntegration/java/com/liferay/change/tracking
```
Comment (joshchong) [inline test.properties:7516]: change this to `**/change-tracking/**/CTCollectionServiceTest.java`

**Review by joshchong — CHANGES_REQUESTED** [review]

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by joshchong** [comment]
ci:forward

**Pull request closed without merging**